layers/core_checks/shader_validation.cpp

```diff
@@ -286,6 +286,50 @@ bool CoreChecks::ValidateShaderInputAttachment(const SHADER_MODULE_STATE &module
     return skip;
 }
 
+bool CoreChecks::ValidateConservativeRasterization(const SHADER_MODULE_STATE &module_state, const Instruction &entrypoint,
+                                                   const PIPELINE_STATE &pipeline) const {
+    bool skip = false;
```

Comment:
You can early out of this function if `conservativeRasterizationPostDepthCoverage` is true. Which avoids executing most of this function.

```suggestion
    bool skip = false;
    
    const bool foobar = phys_dev_ext_props.conservative_rasterization_props.conservativeRasterizationPostDepthCoverage;
    if (foobar) {
        return skip;
    }
```